plugins/Analyzer/AnalyzerWidget.cpp

```diff
@@ -17,29 +17,32 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 #include "AnalyzerWidget.h"
-#include "edb.h"
+#include "Function.h"
 #include "IAnalyzer.h"
 #include "IDebugger.h"
+#include "IRegion.h"
 #include "MemoryRegions.h"
 #include "State.h"
-#include "IRegion.h"
-#include "Function.h"
+#include "edb.h"
 #include <QAbstractScrollArea>
 #include <QDebug>
+#include <QDir>
+#include <QElapsedTimer>
 #include <QFontMetrics>
 #include <QMouseEvent>
 #include <QPainter>
-#include <QScrollBar>
-#include <QDir>
-#include <QElapsedTimer>
 #include <QPixmap>
+#include <QScrollBar>
 
 namespace AnalyzerPlugin {
 
-//------------------------------------------------------------------------------
-// Name:
-//------------------------------------------------------------------------------
-AnalyzerWidget::AnalyzerWidget(QWidget *parent, Qt::WindowFlags f) : QWidget(parent, f)  {
+/**
+ * @brief AnalyzerWidget::AnalyzerWidget
```

Comment:
Don't you find that such placeholders that contain no substance, just repeating the obvious, are useless? I think it'd be better to create them whenever you're really going to make useful comments (document the function), otherwise these are just like dust in the source code — useless but unavoidable (besides, Style Guide doesn't appear to require them, so they seem actually avoidable).

Reply:
So, at least for me, they serve 2 primary purposes.

1. While not immediately having substance, they do provide a designated place to put any information we need to know about the function, should we need it. It's a consistent place to put it.

2. but more importantly, I really like having a visual break between functions that is unmistakable, it helps a lot with navigation when scrolling to have a different colored comment block dividing up functions so they don't blend together. I loved the style I learned in college with the wall of dashes, but my editor does Doxygen comments in like an instant, so it's hard to argue with that level of convinience.